src/web/Errors.ts

```diff
@@ -1,8 +1,5 @@
-/* eslint-disable eslint-comments/no-unlimited-disable */
-/* eslint-disable */
-// @ts-nocheck TODO(TS) provide types
 export class GesturePropError extends Error {
-  constructor(name, value, expectedType) {
+  constructor(name: string, value: any, expectedType: string) {
```

Comment:
`any`

Reply:
We need types `web/FlingGestureHandler` for me to know what type can it have, `unknown` is not a good option since it is later used in a comment.

Reply:
it's `number`, I left some comments for myself so don't worry about that.